install_config/upgrading/manual_upgrades.adoc

```diff
@@ -899,6 +899,142 @@ For Pacemaker HA multiple master deployments:
 
+[[manual-upgrading-efk-logging-stack]]
+== Upgrading the EFK Logging Stack
+
+If you have previously link:../../install_config/aggregate_logging.html[deployed
+the EFK logging stack] and want to upgrade to the latest logging component
+images, you must take the following steps to safely upgrade with minimal
+disruption to your log data.
+
+[NOTE]
+====
+The following steps apply when upgrading from {product-title}
+ifdef::openshift-origin[]
+1.1 to 1.2.
+endif::[]
+ifdef::openshift-enterprise[]
+3.1 to 3.2.
+endif::[]
+====
+
+. Ensure you are working in the project where the EFK stack was previously
+deployed, and stay in that project for the remainder of these steps. For
+example, if the project is named *logging*:
++
+----
+$ oc project logging
+----
+
+. Scale down your Fluentd instances to 0:
++
+----
+$ oc scale dc/logging-fluentd --replicas=0
+----
++
+Wait until they have properly terminated. This helps prevent loss of data by
+giving them time to properly flush their current buffer and send any logs they
+were processing to Elasticsearch.
+
+. Scale down your Kibana instances:
++
+----
+$ oc scale dc/logging-kibana --replicas=0
+----
++
+If you have an operations deployment, also run:
++
+----
+$ oc scale dc/logging-kibana-ops --replicas=0
+----
+
+. Once your Fluentd and Kibana pods are confirmed to be terminated, you can
+safely scale down the Elasticsearch pods:
++
+----
+$ oc scale dc/logging-es-<unique_name> --replicas=0
+----
++
+If you have an operations deployment, also run:
++
+----
+$ oc scale dc/logging-es-ops-<unique_name> --replicas=0
+----
+
+. After your Elasticsearch pods are confirmed to be terminated, pull in the
+latest EFK images using the same procedure described in
+link:../../install_config/upgrading/manual_upgrades.html#importing-the-latest-images[Importing
+the Latest Images], replacing the *openshift* project with the project where the
+EFK stack was previously deployed.
++
+For example, if the project is named *logging*:
++
+----
+$ oc import-image -n logging <imagestream>
+----
++
+The list of image streams are:
++
+----
+logging-auth-proxy
+logging-elasticsearch
+logging-fluentd
+logging-kibana
+----
+
+. With the latest images in your repository, you can now rerun the deployer to
+generate any missing or changed features.
+
+.. First, ensure that your OAuth client has been deleted:
++
+----
+$ oc delete oauthclient --selector logging-infra=support
+----
+
+.. Then, proceed to follow the same steps as done previously in
+link:../../install_config/aggregate_logging.html#deploying-the-efk-stack[Deploying
+the EFK Stack]. After the deployer completes, re-attach your persistent volumes
+you were using previously.
+
+. Next, scale Elasticsearch back up incrementally so that the cluster has time
+to rebuild.
+
+.. To begin, scale up to 1:
++
+----
+$ oc scale dc/logging-es-<unique_name> --replicas=1
+----
++
+Tail the logs of the resulting pod to ensure that it was able to recover its
```

Comment:
As a novice user, tail is a new-to-me term. Should a command be added here to show what you mean, or do you think the target audience should already know what is meant here?